website/src/hugo/content/changelog.md

```diff
@@ -8,6 +8,35 @@ Maintenance branches are merged before each new release. This change log is
 ordered chronologically, so each release contains all changes described below
 it.
 
+# v0.21.2 (2020-03-20)
+
+## Enhancements
+
+* [#3196](https://github.com/http4s/http4s/pull/3196): Add convenience functions to `Caching` middleware. 
+
+## Noteworthy
+
+* [#3253](https://github.com/http4s/http4s/pull/3253): Preparation for Dotty support. Should be invisible to end users, but calling out because it touches a lot. 
+p
+## Dependency updates
+
+* cats-2.1.1
```

Comment:
There are some missing dependency updates here.

Reply:
I usually don't mention the test-only dependencies, but I could if you think they're insightful.

Reply:
I was thinking of the different version bumps of as mentioned in the 0.20.x docs below. Putting on "just reading the release docs" glasses,  I didnt find it entirely obvious that they are also included in the 0.21.x release as well.

Reply:
Ah, yeah.  That's always been awkward to express given the interleaved branches.  We have that note at the top that you should always read top down until the version you're coming from.  I don't know how to make it clear without being repetitive.

Reply:
Me neither. so I guess this as good as we can get atm.